src/test/java/org/cactoos/http/HtCookiesTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.http;
+
+import org.cactoos.io.InputOf;
+import org.cactoos.text.JoinedText;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link HtCookies}.
+ *
+ * @author Vseslav Sekorin (vssekorin@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public final class HtCookiesTest {
+
+    @Test
+    public void takesCookiesOfHttpResponse() {
+        MatcherAssert.assertThat(
+            new HtCookies(
+                new HtHead(
+                    new InputOf(
+                        new JoinedText(
+                            "\r\n",
+                            "HTTP/1.1 200 OK",
+                            "Content-type: text/plain",
+                            "Set-Cookie: path=/; domain=.google.com",
+                            "",
+                            "Hello, dude!",
+                            "How are you?"
+                        )
+                    )
+                )
+            ).get("domain"),
+            Matchers.equalTo(".google.com")
```

Comment:
@VsSekorin It's not a big issue, but `org.hamcrest.Matchers#equalTo` just creates a new instance of `org.hamcrest.core.IsEqual` inside. So, let's replace using of static `equalTo` method with instantiating a new instance of `IsEqual`, please? It will decrease usage of static methods in this class.